gocal.go

```diff
@@ -302,6 +318,8 @@ func (gc *Gocal) parseEvent(l *Line) error {
 		gc.buffer.Categories = strings.Split(l.Value, ",")
 	case "URL":
 		gc.buffer.URL = l.Value
+    case "COMMENT":
```

Comment:
This case statement is not indented properly, please edit and this can be merged.

Reply:
Reformatted code.